google-cloud-storage/src/main/java/com/google/cloud/storage/Crc32cUtility.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.storage;
+
+class Crc32cUtility {
+  private Crc32cUtility() {}
+
+  /*
+     Implementation borrowed taken from: https://github.com/google/crc32c/blob/main/src/crc32c_portable.cc#L16-L59
+  */
+
+  private static final int kByteExtensionTable[] =
```

Comment:
Java style wants the array brackets on the type not the variable.

```suggestion
  private static final int[] kByteExtensionTable =
```